en/components/general/wpf_to_angular_guide/wpf_to_angular_guide.md

```diff
@@ -0,0 +1,13 @@
+---
+title: WPF to Angular guide
+_description: WPF to Angular guide topics help you switch from WPF to Angular.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular
+---
+
+## WPF to Angular guide
+
+<div class="divider--half"></div>
```

Comment:
This is an empty page - add some content or links to the related topics

Reply:
Addressed.